model/metrics/container.yaml

```diff
@@ -14,6 +14,20 @@ groups:
         brief: "The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels."
         requirement_level: opt_in
 
+  - id: metric.container.cpu.usage
```

Comment:
what would be the reason to record it as a `container.cpu.usage` and not one of the existing metrics in `system.cpu.*`? 

I understand there are other metrics in `container` namespace already, and it's be nice to clarify whether we need different ones and if we do, then document why.

Reply:
It is documented at https://github.com/open-telemetry/semantic-conventions/blob/v1.26.0/docs/system/system-metrics.md?plain=1#L15-L17.

Reply:
thanks! would you mind adding a similar statement to container metrics (that they are not the same as system ones)?

do you know why - would system metrics be different than container metrics? or it's a way to distinguish them if both sets of metrics are reported?

Reply:
Mostly a way to distinguish them since a container (or Pod) is not a "system"(=="host") neither a standalone process (it's a group of processes, aka cgroup). I wouldn't be surprised if some API specific metrics might differ and extra transformations would be needed to make them match the `system.*` metrics, so that would be an extra pain/hassle for the implementations I guess. Also it depends from which point of view we observe a container. If we run a collector inside a container treating it as a host then from this point of view the container is a "host" and hence `system.*` metrics should be collected using the `hostmetrics` receiver. When we observe containers/pods from outside, using a centralized API, we treat them mostly as "processes" (but strictly speaking a container can include more than 1 process). Since those entities are technology specific, we decided back then that for centralized metrics that are collected from outside of the resource's scope (ie container metrics collected centrally for a k8s cluster) we use resource specific namespace like `container.*` or `k8s.pod`.
I think this is somehow similar to the `jvm.*` namespace.

Some relevant discussions, issues:
- https://github.com/open-telemetry/semantic-conventions/issues/226#issuecomment-1700529192
- https://github.com/open-telemetry/opentelemetry-specification/pull/2388#issuecomment-1057120767

Reply:
thank you for the context!